src/pepr/operator/controllers/config/config.ts

```diff
@@ -266,16 +280,8 @@ async function handleCABundleUpdate(caBundle: ConfigCABundle, updateClusterResou
 
   UDSConfig.caBundle.includeDoDCerts = caBundle.includeDoDCerts === true;
   UDSConfig.caBundle.includePublicCerts = caBundle.includePublicCerts === true;
-
-  if (caCertsConfigMap && caCertsConfigMap.data) {
-    if (UDSConfig.caBundle.includeDoDCerts && caCertsConfigMap.data["dodCACerts"]) {
-      UDSConfig.caBundle.dodCerts = caCertsConfigMap.data["dodCACerts"];
-    }
-
-    if (UDSConfig.caBundle.includePublicCerts && caCertsConfigMap.data["publicCACerts"]) {
-      UDSConfig.caBundle.publicCerts = caCertsConfigMap.data["publicCACerts"];
-    }
-  }
+  UDSConfig.caBundle.dodCerts = dodCerts;
+  UDSConfig.caBundle.publicCerts = publicCerts;
```

Comment:
Just checking my understanding, previously this was conditional on the `include*` toggles, but that was unnecessary because we check those values "later" when actually making/updating configmaps, is that correct?